### PR TITLE
 Provide API for getting webclip list 

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] #1365
[Problem] No API to get list of webclips
[Solution] Provide API

### CODE REVIEW

**Comment by lslachciak** [comment]
Running server:
cd examples/mobile/HomeAppServer
npm install
npm start

testing API
open url:
http://localhost:3000/api/webcliplist

note some items should be available in webclip directory

sample response

["webclip/restaurant","webclip/weather"]

**Comment by singa2000** [comment]
@lslachciak 
There is not needed the express server in the examples/mobile/HomeAppServer
The express server is located another service module.
examples/mobile/HomeAppServer is just for your testing?

In TAU, just assuming that a new folder will be added in the examples/mobile/HomeApp/webclip folder.
When the new folder is added , please add the web clip card in the HomeScreen.

**Comment by lslachciak** [comment]
@singa2000 this PR is for testing HomeApp.
HomeApp can be used with any other web server which provide API:
api/webcliplist

File: examples/mobile/HomeAppServer/bin/www [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+var app = require('../app');
+var debug = require('debug')('homeappserver:server');
+var http = require('http');
+
+/**
+ * Get port from environment and store in Express.
+ */
+
+var port = normalizePort(process.env.PORT || '3000');
```
Comment (DohyungLim) [inline examples/mobile/HomeAppServer/bin/www:15]: Is the 3000 port fixed?

**Review by DohyungLim — COMMENTED** [review]
1 comment.

File: examples/mobile/HomeAppServer/bin/www (hunk 1 above)
Comment (lslachciak) [reply examples/mobile/HomeAppServer/bin/www:15]: it is default port used by express server
You can change port when starting server using PORT env variable e.g.
`PORT=8000 npm start`

**Review by TomaszLukawskiSam — APPROVED** [review]
The patch works as expected. Looks good and works good.

**Review by singa2000 — APPROVED** [review]
LGTM

**Pull request merged**